challenge.rb

```diff
@@ -21,11 +32,14 @@ def is_of_age? age, minimum_age
 # This question will be graded independently of question one.
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
-def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+def okay_to_drink?(age, drinking_age)
+  if is_of_age?(age, drinking_age) == true
```

Comment:
This is a little bit redundant - you could have just written `if is_of_age?(age, drinking_age)`